src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -493,8 +493,9 @@ static int do_map()
     }
 
     if (forker.is_parent()) {
+      global_init_postfork_start(g_ceph_context);
```

Comment:
@dillaman oic! i will try to run the rbd tests next time when i am touching rbd related code.